gravitee-apim-console-webui/src/management/api/notifications/notifications-list/notofication-details/notification-details.component.html

```diff
@@ -15,4 +15,32 @@
     limitations under the License.
 
 -->
-🚧 Notification detail component 🚧
+<div>
+  <span class="md-headline">
+    <gio-go-back-button [ajsGo]="{ to: 'management.apis.notifications-ng' }"></gio-go-back-button>
+    Update <b>{{ notificationSettings.name }}</b> notification settings
+    <span *ngIf="notifier" class="gio-badge-neutral">{{ notifier.name }}</span>
+  </span>
+  <form autocomplete="off" gioFormFocusInvalid [formGroup]="notificationForm" (ngSubmit)="onSubmit()" *ngIf="!isLoadingData">
+    <mat-card>
+      <h3>Which events would you like to subscribe for?</h3>
+
+      <div>Configure the notifier you want to use</div>
+      <mat-form-field class="settings-general__form__card__form-field">
```

Comment:
ha copy past 🐛 .
As you can see we try to use the [BEM](https://getbem.com/) naming convention as much as possible. 
It's not mandatory, but it's better to get as close as possible.

Reply:
Fixed